example/src/main/java/com/alibaba/nacos/example/NamingExample.java

```diff
@@ -32,6 +32,8 @@
 
 /**
  * Nacos naming example.
+ * <p>Add the JVM parameter to run the NamingExample:</p>
+ * {@code -DserverAddr=127.0.0.1:8848 -Dnamespace=dev}
```

Comment:
If want to hint. I think use parameters is right. because others may not same as you.

change to `-DserverAddr=${nacos.server.ip}:${nacos.server.port} -Dnamespace=${namespaceId}` will be better.

Reply:
yes, that a good idea. I will change it later.